### ISSUE TITLE
Duration Check / Deltas?

### ISSUE THREAD

**Issue by arrmo** [BrettSheleski/comchap#17]
Hi,

Just a thought, but any feeling for the timing "error" that should exist when cutting a file? Asking because I trimmed a file, and found that the deltas in the EDL file, vs. the file size change was off by a few seconds. Not a big deal of course, but just a bit surprised that it exists at all really.

Thoughts?

Thanks!

**Comment by BrettSheleski** [comment]
Are you saying the total time specified by the edl file is different from the resulting file?

**Comment by arrmo** [comment]
Not quite. If I take the original file size, subtract out the commercial time (from EDL) - the output file is not quite the same number as this (example below). Not a biggie, but trying to understand it.

```
> Original File Length (secs): 14819.739422
> Total Commercial Duration (cut from file, secs): 3755.23 (25.3%)
> Expected Final File Length (secs): 11064.509422
> Actual Final File Length (secs): 11062.123578
> Error in Final File Length (secs): -2.385844 (ABS = 2.38584)
```

Thanks!

**Comment by BrettSheleski** [comment]
I'll have to see what's going on with it.  My guess is ffmpeg cannot create parts at the exact times specified for some reason.
